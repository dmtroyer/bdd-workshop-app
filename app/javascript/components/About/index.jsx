import React from 'react';
import MainContainer from '../MainContainer';

const About = () => (
  <MainContainer dataTestId="about">
    <h1>Game App</h1>
    <p data-testid="about-content">
      A demo app of BDD built for RailsConf 2019
    </p>
  </MainContainer>
);

export default About;
