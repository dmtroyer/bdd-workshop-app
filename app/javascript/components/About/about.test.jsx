import React from 'react';
import { render, cleanup } from 'react-testing-library';
import About from '.';
import 'jest-dom/extend-expect';

jest.mock('../../hooks/useProfile');

describe('About', () => {
  beforeEach(() => {
    cleanup();
  });

  it('renders the about page', () => {
    const { container } = render(<About />);
    expect(container).toMatchSnapshot();
  });
});
