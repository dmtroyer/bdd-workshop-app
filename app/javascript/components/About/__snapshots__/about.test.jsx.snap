// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders the about page 1`] = `
<div>
  <main
    class="m-5"
    data-testid="about"
  >
    <h1>
      Game App
    </h1>
    <p
      data-testid="about-content"
    >
      A demo app of BDD built for RailsConf 2019
    </p>
  </main>
</div>
`;
